main/webapp/modules/core/scripts/index/default-importing-controller/controller.js

```diff
@@ -265,10 +265,13 @@ Refine.DefaultImportingController.prototype.getPreviewData = function(callback,
           "limit" : numRows || 100 // More than we parse for preview anyway
         }),
         null,
-        function(data) {
-          result.rowModel = data;
-          callback(result);
-        },
+		  function(data) {
```

Comment:
@WaltonG: your spacing seams off a bit. What do you think?

Reply:
True its too indented. Will sort it out.